js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,27 @@
+function countMinimumNumberOfDeletions(string) {
+  if (!string.length) {
+    return 0;
+  }
+  let k = 0;
+  let numberOfDeletions = 0;
+  while (k < string.length - 1) {
+    if (string[k] === "A" && string[k + 1] !== "B") {
```

Comment:
Everything looks fine to me!
although a few suggestions,
1. alternatingCharacters solutions seems be be built around fact that alternating strings in the input array will always be 'A' and 'B'. Could have been better if it worked on any alternating string.
2. braces at line 9,12, 21 could be avoided as it contains only one line.